lib/technical_metadata_service.rb

```diff
@@ -11,14 +11,17 @@
 # Switching to a more granular data model that has file metadata separate from
 # the Work metadata will allow us to simplify this greatly.
 class TechnicalMetadataService
-  # @param [Dor::Item] dor_item The DOR item being processed by the technical metadata robot
+  # @param [String] content_metadata The xml representation of the content metadata from DOR
+  # @param [String] tech_metadata The xml representation of tech metadata from DOR
```

Comment:
`tech_metadata` is expected to be a string but may also be `nil` if the techMD datastream is `#new?`, right? Worth reflecting in the YARD documentation that this `@param` may be either a string or a `nil`?

Reply:
Sure, I can make that change.

Reply:
Thanks, @jcoyne.